ecs-agent/daemonimages/csidriver/Makefile

```diff
@@ -23,14 +23,19 @@ IMAGE_REF?="ebs-csi-driver:latest"
 # caution. See `go help test`
 # unit tests include the coverage profile
 GOTEST=${GO_EXECUTABLE} test -count=1
+ifeq (${TARGET_OS},windows)
+	CSI_DRIVER_BIN=bin/ebs-csi-driver.exe
+else
+	CSI_DRIVER_BIN=bin/ebs-csi-driver
+endif
 
 all: tarfiles/ebs-csi-driver.tar
 
 bin/ebs-csi-driver:
```

Comment:
Can you check if the built binaries are working for Windows platform?

Usually, we use `GOOS=${TARGET_OS}` before the build command.

Reply:
Set GOOS flag for build command in latest.

Reply:
Please use `GOOS=${TARGET_OS}` instead. `GOOS` might not be set on the build platform.

Reply:
Done